src/router/metadata/cap.rest.security.template.ts

```diff
@@ -20,23 +20,50 @@ export default function makeSecurity(authConfig: Auth, hasCORSEnabled: boolean =
                 },
             ],
         };
-        if (authConfig.strategy.oauth) {
+        const { strategy } = authConfig;
+        const { oauthPolicy } = strategy;
```

Comment:
I don't see 'strategy' used anywhere?
```suggestion
        const { oauthPolicy } = authConfig.strategy;
```

Reply:
updated